doc/container-environment.md

```diff
@@ -55,9 +55,9 @@ On top of the standard set of devices, the following are also set up for conveni
 ## Mounts
 The following mounts are set up by default under LXD:
 
- - `/proc` (proc)
- - `/sys` (sysfs)
- - `/sys/fs/cgroup/*` (cgroupfs) (only on kernels lacking cgroup namespace support)
+ - `/proc`
```

Comment:
I think the "(proc)" indicates what filesystem type the mounts uses, so should still be kept I think.

Reply:
Does "procfs" make sense for it then?

Reply:
Nope, `proc` is the filesystem type in this case.